Src/FluentAssertions/Formatting/FormattingOptions.cs

```diff
@@ -1,12 +1,23 @@
-﻿namespace FluentAssertions.Formatting;
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace FluentAssertions.Formatting;
 
 public class FormattingOptions
 {
+    internal static readonly List<IValueFormatter> CustomFormatters = [];
+
+#pragma warning disable SA1401, CA1822
+    internal static List<IValueFormatter> StaticScopedFormatters = [];
```

Comment:
The only reason for this static field is to be able to reach this in `Formatter.Formatters`.

If you have a better way, please enlighten me 🙏 (because this looks like a bad and/or ugly hack..)